.modules/service/variables.tf

```diff
@@ -65,3 +65,21 @@ variable "rolling_updates" {
   default     = false
   type        = bool
 }
+
+variable "create_policies" {
```

Comment:
This should be calculated, there should be no need to pass this and the ARN.

Reply:
I tried it (described in the PR description), and it did not work. I'll check if it works if we pass the role name and use it a data source as you suggested.

Reply:
It looks like it works if a name is passed and used for the data source instead of passing the ARN directly. Thanks for the tip!